### ISSUE TITLE
Another training data

### ISSUE THREAD

**Issue by jackieyung** [hbertiche/PBNS#11]
Hi, @hbertiche Thanks for your solid work. 

I want to try more experiments on a different dataset, but sadly,  I don't know how to make the training dataset, that is Outfit.obj and Outfit_config.mat. Could you share the method about how to generate the training dataset? or provide another more data for me to try. Thanks a lot.

**Comment by hbertiche** [comment]
Thanks for your interest in my work!

To generate new OBJ files you can use the `writeOBJ` function defined in `PBNS/PBNS/IO.py`,
provide the vertex locations as a `N x 3` numpy array and the faces as a list of faces, with each face being a list of vertex indices (a `N x 3` array with triangle faces would also work)
another possibility is to design them manually with a 3D software and export it as OBJ file, although my `readOBJ` function is very basic and might fail with fancy OBJ files, for this option, I recommend using Blender,

then, for the '[XXX]_config.mat' file you have a description in `PBNS/PBNS/README.md` (note this is a specific README.md file for the subfolder 'PBNS', different from the main README.md of the project), in the **Object** section,
as also described there, the methodology still works without providing this file (it will use a default configuration)

I hope this is helpful for you,
please do not hesitate to ask further questions

**Comment by jackieyung** [comment]
Thanks for your reply. I'll try it.

I have another question. I see all your models are training on SMPL person model from your codes. If I want to retarget one cloth to another non-SMPL person model, for example from mixamo.com. How can I do it? Thanks.

**Comment by hbertiche** [comment]
the code in this repo, as it is, works for SMPL model,
it is possible to apply this methodology to any rigged 3D model, as shown in the paper, but you'd need to adapt this code to do so,
more specifically, the `PBNS/PBNS/Data/data.py`, such that it returns the pose (as quaternions or axis-angles), the transformation matrices of each joint as 3x4 arrays and the body vertex locations,

soon we will release something more complete that works with any 3D avatar, we can't say when though,

**Comment by hbertiche** [comment]
as promised, we have released a code similar to this one, that works with arbitrary 3D models,
find it here
https://github.com/hbertiche/NeuralClothSim

this is a bigger contribution that additionally permits learning cloth dynamics unsupervisedly,
